test/testConstants/FinancialDetailsTestConstants.scala

```diff
@@ -1543,58 +1542,60 @@ object FinancialDetailsTestConstants {
 
 }
 
-case class ANewCreditAndRefundViewModel(model: CreditAndRefundViewModel = CreditAndRefundViewModel(List(), None)) {
+case class ANewCreditAndRefundModel(model: CreditsModel = CreditsModel(0.0, 0.0, Nil)) {
 
-  def withAvailableCredit(availableCredit: BigDecimal): ANewCreditAndRefundViewModel = {
-    val balanceDetails = model.balanceDetails.getOrElse(BalanceDetails(0.0, 0.0, 0.0, None, None, None, None, None))
-      .copy(availableCredit = Some(availableCredit))
-    ANewCreditAndRefundViewModel(model.copy(model.creditCharges, Some(balanceDetails)))
+  private def getTaxYear(localDate: LocalDate): Int = {
```

Comment:
yet another generic conversion we can create for TaxYear type: LocalDate => TaxYear